docs/source/code/data.rst

```diff
@@ -34,6 +35,7 @@ Data Pipeline Components
 .. automodule:: flash.core.data.properties
 
 .. automodule:: flash.core.data.process
+    :noindex:
```

Comment:
Without this, it gives the following warning because there are two identical docs, i.e. `flash.core.data.process.Serializer` and `flash.Serializer`.
```
Warning, treated as error:
/home/docs/project/flash/core/model.py:docstring of flash.core.model.Task.serializer:1:more than one target found for cross-reference 'Serializer': flash.core.data.process.Serializer, flash.Serializer
Makefile:20: recipe for target 'html' failed
```